### ISSUE TITLE
[Arcs] Dominant Faith and Blightkin/Edenguard

### ISSUE THREAD

**Issue by ashleonardj** [LederCards/cards#59]
If the Believer has made it to act III it may come to pass that there are more ambitions boxes than ideal markers to put in them. This becomes a problem when The Believer wants to declare Edenguard if all the ideal markers are already in play. What happens in this instance?

**Comment by turmoilbyrd** [comment]
They have to use a 6 or 7 to declare and not a Faithful card

**Comment by turmoilbyrd** [comment]
by RAW, you can declare the Edenguard ambition with Faithful card, but will not place a Ideal marker since there is no more to be placed.

**Comment by jdyearsley** [comment]
Correct, nothing would happen in this case. I'll add an FAQ about this.
